foreman.fc

```diff
@@ -30,7 +30,7 @@
 /var/log/foreman(/.*)?                              gen_context(system_u:object_r:foreman_log_t,s0)
 
 /var/run/foreman(/.*)?                              gen_context(system_u:object_r:foreman_var_run_t,s0)
-/run/foreman\.sock                              --  gen_context(system_u:object_r:foreman_var_run_t,s0)
+/run/foreman\.sock                                  gen_context(system_u:object_r:foreman_var_run_t,s0)
```

Comment:
this also could be `-s`, but then I don't know why it works on EL7…

Reply:
It does? Weird, anyway this is okay I have never seen `-s` in use to be honest.